recipes/aisim/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "aisim" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aisim-{{ version }}.tar.gz
+  sha256: e9a38c63301af8eaac71e2858886d4dec0c9aef686e641db2c4f7f3c6ba42b11
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - setuptools >=38.3.0
```

Comment:
Please remove setuptools here, since you have pip

Reply:
Fixed in 5aaa2dc.